### ISSUE TITLE
How to access the Admin portal?

### ISSUE THREAD

**Issue by subhendukundu** [hodgef/apiker#134]
How do we deploy or access the admin portal?

**Comment by hodgef** [comment]
Hey @subhendukundu,

Generally if you pass the  [`adminPanel: true`](https://hodgef.github.io/apiker/interfaces/Options.html#adminPanel) option in your Apiker constructor, it should register the following Admin panel routes:

https://github.com/hodgef/apiker/blob/70fe6a8f025cc97979aecb1b1b88d9ea522001d2/src/components/Admin/Admin.ts#L15

However, I think the panel feature might be currently broken since I recently changed how logging works (which affects many features of the panel). Before, all logging  was saved in a single Durable Object (State) instance, and now we use an instance per IP which is more performant.

https://github.com/hodgef/apiker/commit/eed129f17b37d1d17a7eff75a70f4cbb4c5b5aae#diff-a7bd43ff2dfeaf5c57db069a318c79b0a48101199d2d7af4a35a5b89aa6f248cR35

Long story short: The basic panel we have now might be broken, so I will mark this as a bug and pick it up very soon.

**Comment by hodgef** [comment]
**Update (2026): the Admin panel got a major overhaul.** 🎉

See this Sandbox capture for an example:

<img width="1660" height="1649" alt="Image" src="https://github.com/user-attachments/assets/446ae892-dd79-4ef6-8154-3b97767568eb" />


For more information, check out the [documentation](https://hodgef.com/apiker/).
